File: src/Inventory.Entities/Articulo.cs
namespace Inventory.Entities.Models;

public class Articulo
{
    public int Id { get; set; }
    public string Codigo { get; set; } = null!;
    public string Descripcion { get; set; } = null!;
    public decimal Precio { get; set; }
    public string? ImagenUrl { get; set; }
    public int Stock { get; set; }

    public ICollection<ArticuloTienda> ArticuloTiendas { get; set; } = new List<ArticuloTienda>();
    public ICollection<ClienteArticulo> ClienteArticulos { get; set; } = new List<ClienteArticulo>();
}

File: src/Frontend/src/app/core/models/articulo.ts
import { TiendaItemDto } from "./tienda";

export interface ArticuloItemDto {
  id: number;
  codigo: string;
  descripcion: string;
  precio: number;
  imagenUrl?: string | null;
  stock: number;
  fecha?: string;
  tiendas: TiendaItemDto[];
}
export interface ArticuloDetailDto extends ArticuloItemDto {}
